ts/Core/Axis/PlotLineOrBand/PlotLineOrBand.ts

```diff
@@ -72,8 +73,30 @@ class PlotLineOrBand {
      * */
 
     public static compose<T extends typeof Axis>(
+        ChartClass: Chart,
         AxisClass: T
     ): ReturnType<typeof PlotLineOrBandAxis.compose> {
+
+        addEvent(ChartClass, 'afterInit', function (): void {
+            this.labelCollectors.push((): SVGElement[] => {
+                const labels: SVGElement[] = [];
+
+                for (const axis of this.axes) {
+
+                    if (!axis.options.labels.allowOverlap) {
```

Comment:
This option is about the axis labels. I wouldn't expect it to affect the plot band/plot line labels. How does this work currently?

What I would expect is an `allowOverlap` option on the `plotBand.label` itself.